File: screenshot.py
from playwright.sync_api import sync_playwright
import time

def run(playwright):
    browser = playwright.chromium.launch(headless=True)
    page = browser.new_page()
    page.set_viewport_size({"width": 1440, "height": 900})
    page.goto('http://localhost:5173')
    time.sleep(3) # Wait for animations/load
    page.screenshot(path='frontend_screenshot.png', full_page=True)
    browser.close()
    print("Screenshot saved.")

with sync_playwright() as playwright:
    run(playwright)

File: scripts/screenshot.js
const puppeteer = require('puppeteer');

(async () => {
  const browser = await puppeteer.launch({ headless: 'new' });
  const page = await browser.newPage();
  await page.setViewport({ width: 1280, height: 800 });
  await page.goto(process.env.FRONTEND_URL || 'http://localhost:5173', { waitUntil: 'networkidle0' });
  await new Promise(resolve => setTimeout(resolve, 2000));
  await page.screenshot({ path: 'docs/frontend_screenshot.png', fullPage: true });
  await browser.close();
  console.log('Screenshot saved to frontend_screenshot.png');
})();
